3.10/indexing-index-basics.md

```diff
@@ -369,13 +369,19 @@ The geo index stores two-dimensional coordinates. It can be created on either tw
 separate document attributes (latitude and longitude) or a single array attribute that
 contains both latitude and longitude. Latitude and longitude must be numeric values.
 
+Furthermore, a geo index can also index GeoJSON objects. GeoJSON is a
```

Comment:
"... a geo index can also index starndard GeoJSON objects" (and the link on GeoJSON).
"GeoJSON uses JSON syntax to describe geomet..."

Reply:
Done.